### PR TITLE
bug/WDP190703-6_crossed_price

### PR DESCRIPTION
**Pull request by kamil-polanski**
Add div's with crossed price acc to task

### CODE REVIEW

File: src/partials/50_section-products.html [hunk 1]
```diff
@@ -46,7 +46,8 @@ <h5>Aenean Ru Bristique</h5>
               <a href="#" class="btn-outline"><i class="far fa-heart"></i></a>
               <a href="#" class="btn-outline"><i class="fas fa-exchange-alt"></i></a>
             </div>
-            <div class="price"><div class="btn-main-small no-hover">$ 30.00</div></div>
+            <div class="price btn-main-small no-hover"><s></s></div>
```
Comment (thisisui) [inline src/partials/50_section-products.html]: do we really need this part as this is empty, on other lines you are not using <s> tag so probably it is not needed. I would rather remove parts of HTML which are not giving any visual representation on the page like this empty div tag

Comment (kamil-polanski) [reply src/partials/50_section-products.html]: Well, probably not. I removed empty div's acc to your suggestion.

File: src/sass/components/_product-box.scss [hunk 2]
```diff
@@ -2,7 +2,6 @@
   background-color: #ffffff;
   border: 1px solid #e1e1e1;
   margin-bottom: 2rem;
-
```
Comment (matwjablonski) [inline src/sass/components/_product-box.scss]: why these empty lines are removed?

File: src/sass/components/_product-box.scss [hunk 3]
```diff
@@ -72,11 +63,13 @@
...
     }
   }
-
   .actions {
     padding: 15px 10px 10px 10px;
     display: flex;
     justify-content: space-between;
     align-items: center;
   }
+  .price {
```
Comment (matwjablonski) [inline src/sass/components/_product-box.scss]: add empty line above

File: src/partials/50_section-products.html [hunk 4]
```diff
@@ -46,7 +46,8 @@ <h5>Aenean Ru Bristique</h5>
               <a href="#" class="btn-outline"><i class="far fa-heart"></i></a>
               <a href="#" class="btn-outline"><i class="fas fa-exchange-alt"></i></a>
             </div>
-            <div class="price"><div class="btn-main-small no-hover">$ 30.00</div></div>
+
```
Comment (thisisui) [inline src/partials/50_section-products.html]: please remove empty line

File: src/partials/50_section-products.html [hunk 5]
```diff
@@ -102,7 +104,8 @@ <h5>Aenean Ru Bristique</h5>
...
```
Comment (thisisui) [inline src/partials/50_section-products.html]: to remove

File: src/partials/50_section-products.html [hunk 6]
```diff
@@ -130,7 +133,8 @@ <h5>Aenean Ru Bristique</h5>
...
```
Comment (thisisui) [inline src/partials/50_section-products.html]: to remove

File: src/partials/50_section-products.html [hunk 7]
```diff
@@ -158,7 +162,8 @@ <h5>Aenean Ru Bristique</h5>
...
```
Comment (thisisui) [inline src/partials/50_section-products.html]: to remove

File: src/partials/50_section-products.html [hunk 8]
```diff
@@ -186,7 +191,8 @@ <h5>Aenean Ru Bristique</h5>
...
```
Comment (thisisui) [inline src/partials/50_section-products.html]: to remove

File: src/partials/50_section-products.html [hunk 9]
```diff
@@ -214,7 +220,8 @@ <h5>Aenean Ru Bristique</h5>
...
```
Comment (thisisui) [inline src/partials/50_section-products.html]: to remove

File: src/partials/50_section-products.html [hunk 10]
```diff
@@ -242,7 +249,8 @@ <h5>Aenean Ru Bristique</h5>
...
```
Comment (thisisui) [inline src/partials/50_section-products.html]: to remove

**Comment by matwjablonski** [comment]
please resolve conflicts

File: src/sass/components/_product-box.scss (hunk 2 above)
Comment (kamil-polanski) [reply src/sass/components/_product-box.scss]: That was problem with editor. On save it clears all blank lines. Already I've fixed it.

File: src/partials/50_section-products.html (hunk 4 above)
Comment (kamil-polanski) [reply src/partials/50_section-products.html]: I removed blank lines.

**Pull request merged**